packages/client/hmi-client/src/components/workflow/ops/simulate-ciemss/tera-simulate-ciemss-drilldown.vue

```diff
@@ -5,79 +5,90 @@
-								id="2"
-								v-model="timespan.start"
-								inputId="integeronly"
-								@update:model-value="updateState"
-							/>
-						</div>
+		<!-- Wizard -->
+		<section :tabName="DrilldownTabs.Wizard" class="wizard">
+			<tera-slider-panel v-model:is-open="isSidebarOpen" header="Simulation settings" content-width="420px">
+				<template #content>
+					<div class="toolbar">
+						<p>Click Run to start the simulation.</p>
+						<span class="flex gap-2">
+							<tera-pyciemss-cancel-button :simulation-run-id="cancelRunId" />
+							<Button label="Run" icon="pi pi-play" @click="run" :disabled="showSpinner" />
+						</span>
+					</div>
+					<div class="form-section" v-if="isSidebarOpen">
+						<!-- Presets -->
 						<div class="label-and-input">
-							<label for="3">End time</label>
-							<tera-input-number
-								id="3"
-								v-model="timespan.end"
-								inputId="integeronly"
-								@update:model-value="updateState"
+							<label for="4">Preset (optional)</label>
+							<Dropdown
+								v-model="presetType"
+								placeholder="Select an option"
+								:options="[CiemssPresetTypes.Fast, CiemssPresetTypes.Normal]"
+								@update:model-value="setPresetValues"
 							/>
 						</div>
-					</div>
 
-					<!-- Presets -->
-					<div class="label-and-input">
-						<label for="4">Preset (optional)</label>
-						<Dropdown
-							v-model="presetType"
-							placeholder="Select an option"
-							:options="[CiemssPresetTypes.Fast, CiemssPresetTypes.Normal]"
-							@update:model-value="setPresetValues"
-						/>
-					</div>
-
-					<!-- Number of Samples & Method -->
-					<div class="input-row mt-3">
-						<div class="label-and-input">
-							<label for="4">Number of samples</label>
-							<tera-input-number
-								id="4"
-								v-model="numSamples"
-								inputId="integeronly"
-								:min="1"
-								@update:model-value="updateState"
-							/>
+						<!-- Start & End -->
+						<div class="input-row">
+							<div class="label-and-input">
+								<label for="2">Start time</label>
+								<tera-input-number
+									id="2"
+									v-model="timespan.start"
+									inputId="integeronly"
+									@update:model-value="updateState"
+								/>
+							</div>
+							<div class="label-and-input">
+								<label for="3">End time</label>
+								<tera-input-number
+									id="3"
+									v-model="timespan.end"
+									inputId="integeronly"
+									@update:model-value="updateState"
+								/>
+							</div>
 						</div>
-						<div class="label-and-input">
-							<label for="5">Method</label>
-							<Dropdown
-								v-model="method"
-								:options="[CiemssMethodOptions.dopri5, CiemssMethodOptions.euler]"
-								@update:model-value="updateState"
-							/>
+
+						<!-- Number of Samples & Method -->
+						<div class="input-row">
+							<div class="label-and-input">
+								<label for="4">Number of samples</label>
+								<tera-input-number
+									id="4"
```

Comment:
Why are we doing id=1, id=2, id=3, id=4 What are they meant to do??

Reply:
I don't know, that wasn't me. It only looks like me because I indented it

Reply:
Okay I am going to remove, cause these are causing warnings:

```
The label's for attribute doesn't match any element id. This might prevent the browser from correctly autofilling the form and accessibility tools from working correctly.
To fix this issue, make sure the label's for attribute references the correct id of a form field.
```